README.md

```diff
@@ -78,11 +80,26 @@ The Editing Working Group meets remotely at 9:00AM PST, on the second Friday of
 
 The upcoming schedule is below:
 
-* January 14, 2022
-* Feburary 11, 2022
+* TBD (special meeting for continuing discussion on sanitized/unsanitized clipboard content)
+* Feburary 10, 2022
```

Comment:
@travisleithead Is there a specific reason for why we spell February that way?

Reply:
ha ha, no, just a typo...